### DISCUSSION TITLE
Is it posible to get hardware decoding working on RK3568

### DISCUSSION THREAD

**Discussion by jgerits** [blakeblackshear/frigate#4542] [Ask A Question]
I have a rock pi 3a with a coral m.2 in it. I want to use this for frigate but I can't seem to get hardware decoding to work on this chip. I have already googled a lot and found that I have to recompile ffmpeg myself. I followed the tutorials but I still can't get it working.
Is there anywhere that I can find a way to get this working. 
I tried: https://github.com/blakeblackshear/frigate/pull/1814#issuecomment-939294837 but it still used 100% cpu (1 core) so that is not usable at all.

**Comment by thinhlx1993** [comment]
Sorry, Did you get the answer @zelda1link3 ?

**Reply by halabut69** [reply]
HW Acceleration on the RK3568 should work with the -rk image as described in the docs. If you have issues try asking in the rockchip discussion thread:
https://github.com/blakeblackshear/frigate/discussions/11502
